### DISCUSSION TITLE
I want to change some UI and rebuild code-server. How can i do this?

### DISCUSSION THREAD

**Discussion by slion-L** [coder/code-server#5499] [Q&A] [answered]
I want to change some UI and rebuild code-server. How can i do this?

**Comment by jsjoeio** [comment, marked as answer]
Depends on which parts you want to change! What are you trying to do?

**Reply by slion-L** [reply]
I want to change code in 'lib/vscode'. How can I rebuild?

**Reply by jsjoeio** [reply]
You'll need to create a [new patch](https://coder.com/docs/code-server/latest/CONTRIBUTING#patching-code). Then apply the patch with `quilt` and rebuild with `yarn watch` (if in dev)
